dutch/bos/ontvoering/AllesKomtGoed/alleskomtgoed.md

```diff
@@ -0,0 +1 @@
+Je wordt de volgende dag met een kater wakker in [de hei](../../welkom.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.